models/issue.go

```diff
@@ -69,6 +70,12 @@ type Issue struct {
 var (
 	issueTasksPat     *regexp.Regexp
 	issueTasksDonePat *regexp.Regexp
+
+	// issueNumericPattern matches string that references to a numeric issue, e.g. #1287
+	issueNumericPattern = regexp.MustCompile(`(?:\s|^|\(|\[)(#[0-9]+)(?:\s|$|\)|\]|:|\.(\s|$))`)
```

Comment:
Can this be reused?

Reply:
Mmmm... it might, but I think it could be dangerous to do that. This is because the text to parse here is markdown source, while other parts of the code use that regexp for HTML. It so happens that the same regexp can be used in both formats for this particular case, but that could easily change.
What I'd do (but in a different PR) is to consolidate all kinds of pattern matching related to issue reference in a separate module that deals with markdown, HTML, commit messages, etc. so all of them are in one place.

Reply:
ok